libmachine/drivers/plugin/register_driver.go

```diff
@@ -29,8 +28,6 @@ Please use this plugin through the main 'docker-machine' binary.
 		os.Exit(1)
 	}
 
-	libmachine.SetDebug(true)
```

Comment:
Like noted above, if we simply set `log.IsDebug` to true, the main process is streaming the STDOUT and STDERR from the process and `log.Debug`ing them anyway.